sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/profile/AzureProfile.java

```diff
@@ -20,6 +20,12 @@
 
     /**
      * Creates AzureProfile instance with Azure environment. The global environment is {@link AzureEnvironment#AZURE}.
+     * The tenant ID and subscription ID can be set via environment variables. The environment variables are expected
```

Comment:
@weidongxu-microsoft The `configuration` is used when the user might prefer [EnvironmentCredential](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/identity/azure-identity/src/main/java/com/azure/identity/EnvironmentCredential.java#L48). Instead of making configuration settable, I prefer following the way environment credential handle configuration. At least for credential related, the user only need to set environment variables in their system.

Reply:
Like I said, I prefer user knows when configure from env is used, when it is not. That is the reason of the existence of `EnvironmentCredential` class at first place. It let user use the credential from env consciously.